src/utils/runtime/src/lib.rs

```diff
@@ -25,6 +25,12 @@ use tracing_subscriber::filter;
 use tracing_subscriber::layer::SubscriberExt;
 use tracing_subscriber::prelude::*;
 
+/// Dump logs of all SQLs, i.e., tracing target `pgwire_query_log` to `.risingwave/log/query.log`.
+///
+/// Changing the level of `pgwire` to `TRACE` in `configure_risingwave_targets_fmt` can also turn on
+/// the logs, but without a dedicated file.
+const ENABLE_QUERY_LOG_FILE: bool = false;
```

Comment:
So this now should be set manually by people?

Reply:
Yes. Just like modifying log levels.

Reply:
It's not very flexible but it's currently enough for debugging and diagnosis? 🤔

Reply:
I just run into a use case that would be much easier to debug with this query log. Can you please expose it as a config?